modules/aws-eks-iam-role-group/outputs.tf

```diff
@@ -31,7 +31,7 @@ output "kubernetes_namespace" {
 output "aws_auth_entry" {
   description = "An IAM-Kubernetes binding that has to be put to aws-auth ConfigMap."
   value = {
-    rolearn  = var.iam_path == null ? aws_iam_role.iam_role.arn : replace(replace(aws_iam_role.iam_role.arn, var.iam_path, ""), "/\\/{2,}/", "/")
+    rolearn  = var.iam_path == null ? aws_iam_role.iam_role.arn : replace(aws_iam_role.iam_role.arn, "/${var.iam_path}/", "/")
```

Comment:
Is this just a simplification? I remember that EKS didn't respect IAM paths, so good that replace is still here. 👍🏻

Reply:
It is fix, because it in some cases it returns incorrect `rolearn` value like: 

```
"rolearn" = "arn:aws:iam::9999999999:role/-admin"
```

The case:
- name is `some-tool`
- `var.iam_path` is like `/some-tool/`
- ` aws_iam_role.iam_role.arn` is like `arn:aws:iam::9999999999:role/some-tool/some-tool-admin`